app/components/navigation_header/index.tsx

```diff
@@ -125,12 +125,14 @@ const NavigationHeader = ({
                     hideHeader={hideHeader}
                     theme={theme}
                     topStyle={searchTopStyle}
+                    ref={ref}
                 />
                 }
             </Animated.View>
         </>
     );
-};
+});
 
+NavigationHeader.displayName = 'NavHeader';
```

Comment:
Just curiosity. Is this still needed? What for? (I guess it was some requirement for the ref, but still curious)

Reply:
This is only due to this eslint rule https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md